### PR TITLE
Upgrade xorm to latest to fix insert issue bug

### PR DESCRIPTION
**Pull request by lunny**
Also added an unit tests for newissue to confirm some special characters will not break the creating of issue.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -824,3 +824,5 @@ xorm.io/builder v0.3.6 h1:ha28mQ2M+TFx96Hxo+iq6tQgnkC9IZkM6D8w9sKHHF8=
 xorm.io/builder v0.3.6/go.mod h1:LEFAPISnRzG+zxaxj2vPicRwz67BdhFreKg8yv8/TgU=
 xorm.io/core v0.7.0 h1:hKxuOKWZNeiFQsSuGet/KV8HZ788hclvAl+7azx3tkM=
 xorm.io/core v0.7.0/go.mod h1:TuOJjIVa7e3w/rN8tDcAvuLBMtwzdHPbyOzE6Gk1EUI=
+xorm.io/core v0.7.2-0.20190928055935-90aeac8d08eb h1:msX3zG3BPl8Ti+LDzP33/9K7BzO/WqFXk610K1kYKfo=
```
Comment (guillep2k) [inline go.sum:827]: Why is xorm.io/core v0.7.0 still referenced? Some dependency needs updating too, it seems.

File: models/issue_test.go [hunk 2]
```diff
@@ -320,3 +320,36 @@ func TestIssue_SearchIssueIDsByKeyword(t *testing.T) {
...
+
+func testInsertIssue(t *testing.T, title, content string) {
+	repo := AssertExistsAndLoadBean(t, &Repository{ID: 1}).(*Repository)
+	user := AssertExistsAndLoadBean(t, &User{ID: 2}).(*User)
+
+	var issue = Issue{
+		RepoID:   repo.ID,
+		PosterID: user.ID,
+		Title:    "my issue1",
+		Content:  "special issue's comments?",
```
Comment (guillep2k) [inline models/issue_test.go]: Shouldn't those be `title` and `content` parameters here instead?

File: vendor/github.com/go-xorm/xorm/go.sum [hunk 3]
```diff
@@ -162,3 +166,7 @@ xorm.io/builder v0.3.6 h1:ha28mQ2M+TFx96Hxo+iq6tQgnkC9IZkM6D8w9sKHHF8=
 xorm.io/builder v0.3.6/go.mod h1:LEFAPISnRzG+zxaxj2vPicRwz67BdhFreKg8yv8/TgU=
 xorm.io/core v0.7.0 h1:hKxuOKWZNeiFQsSuGet/KV8HZ788hclvAl+7azx3tkM=
 xorm.io/core v0.7.0/go.mod h1:TuOJjIVa7e3w/rN8tDcAvuLBMtwzdHPbyOzE6Gk1EUI=
+xorm.io/core v0.7.1 h1:I6x6Q6dYb67aDEoYFWr2t8UcKIYjJPyCHS+aXuj5V0Y=
+xorm.io/core v0.7.1/go.mod h1:jJfd0UAEzZ4t87nbQYtVjmqpIODugN6PD2D9E+dJvdM=
```
Comment (guillep2k) [inline vendor/github.com/go-xorm/xorm/go.sum:170]: Again, is it OK to have v0.7.0 and v0.7.1 still referenced here?

**Review by guillep2k — CHANGES_REQUESTED** [review]

File: models/issue_test.go [hunk 4]
```diff
@@ -320,3 +320,36 @@ func TestIssue_SearchIssueIDsByKeyword(t *testing.T) {
...
 }
+
+func testInsertIssue(t *testing.T, title, content string) {
+	repo := AssertExistsAndLoadBean(t, &Repository{ID: 1}).(*Repository)
+	user := AssertExistsAndLoadBean(t, &User{ID: 2}).(*User)
+
+	var issue = Issue{
+		RepoID:   repo.ID,
+		PosterID: user.ID,
+		Title:    "my issue1",
```
Comment (guillep2k) [inline models/issue_test.go]:
```suggestion
		Title:    title,
```

File: models/issue_test.go (hunk 2 above)
Comment (guillep2k) [inline models/issue_test.go]:
```suggestion
		Content:  content,
```

**Review by guillep2k — CHANGES_REQUESTED** [review]

File: go.sum (hunk 1 above)
Comment (sapk) [reply go.sum:827]: It seems `github.com/lafriks/xormstore` need an update.

Comment (sapk) [reply go.sum:827]: @lunny It would maybe cleaner to release a new version for core and xorm than using a commit reference ?

**Comment by lunny** [comment]
@sapk @guillep2k done.

**Review by sapk — APPROVED** [review]
LGTM
I think we can merge this even before `github.com/lafriks/xormstore` is updated (and create an other PR later) or wait for it as you like.

**Review by guillep2k — APPROVED** [review]

**Comment by lunny** [comment]
Should fix #8301

**Comment by lafriks** [comment]
New xormstore version has been tagged

**Pull request merged**